ephios/core/templatetags/user_extras.py

```diff
@@ -78,10 +78,9 @@ def intersects(a, b):
     return bool(set(a) & set(b))
 
 
-@register.filter(name="has_permission")
-def group_has_permission(group, permission):
-    app_label, codename = permission.split(".")
-    return group.permissions.filter(content_type__app_label=app_label, codename=codename).exists()
+@register.filter(name="user_has_permission")
```

Comment:
is there a reason why you don't want to use the standard x in perms check in the template?

Reply:
The perms variable comes from context processors, so that variable is not available in some render contexts.